### PR TITLE
KAA-1397: Update Kaa Sandbox Networking page

### PR DESCRIPTION
**Pull request by Acarus**

### CODE REVIEW

**Review by Slash32 — APPROVED** [review]

**Comment by rasendubi** [comment]
@Acarus this needs update

**Review by Slash32 — CHANGES_REQUESTED** [review]
I would prefer to leave the merged variant in place.
Please drop the TIP change and leave here only the doc/Customization-guide/Code-style/Java/index.md part.

File: doc/Customization-guide/Code-style/Java/index.md [hunk 1]
```diff
@@ -17,9 +17,8 @@ This guide describes tools used to check style and format Java source code.
 1. Go to *File>Settings>IDE Settings>plugins*.
 2. Click on the *Browse repositories* button.
 3. Type *Checkstyle-idea* and install the plugin.
-4. Download the *[configuration for the Google coding conventions](https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml)*.
-5. Go to *File>Settings>IDE Settings>CheckStyle*.
-6. Add the configuration file (`checkstyle.xml`) from root of the Kaa project.
+4. Go to *File>Settings>IDE Settings>CheckStyle*.
+5. Add the configuration file (`checkstyle.xml`) from root of the Kaa project.
```
Comment (rasendubi) [inline doc/Customization-guide/Code-style/Java/index.md]: Where `checkstyle.xml` comes from?

Comment (Acarus) [reply doc/Customization-guide/Code-style/Java/index.md]: `checkstyle.xml` is located in root of the project.  It's google code style rules with some changes.

Comment (rasendubi) [reply doc/Customization-guide/Code-style/Java/index.md]: Oh, ok. I would insert a link here, but that's minor.

**Review by Slash32 — APPROVED** [review]
LGTM

**Comment by rasendubi** [comment]
@Acarus, please, resolve conflicts

**Review by garniy — APPROVED** [review]
LGTM

**Pull request merged**